src/app/pages/PerpetualPage/components/TradeDialog/components/ReviewStep.tsx

```diff
@@ -40,15 +43,30 @@ export const ReviewStep: TransitionStep<TradeDialogStep> = ({ changeTo }) => {
     tradingFee,
   } = analysis;
 
-  const { execute } = usePerpetual_executeTransaction();
+  const { perpetualParameters } = useContext(PerpetualQueriesContext);
+  const [lotSize, lotPrecision] = useMemo(() => {
```

Comment:
We use this in 2 other places, either extract it to a util function or add it to `PerpetualQueriesContext`.